### DISCUSSION TITLE
Help with two overlays

### DISCUSSION THREAD

**Discussion by areknow** [adobe/react-spectrum#1825] [Q&A]
Hello, I have two triggers which open overlays. I would like only one to be open at a time. How can I achieve that?

Use case:
I open dialog 1, then click on the trigger of dialog 2. I would like the first dialog to close so that only 1 dialog is open at a time.

https://codesandbox.io/s/react-aria-useoverlay-with-2-dialogs-vgth9?file=/src/App.js

Thanks!
Arnaud

**Comment by snowystinger** [comment]
Hey! We are working on this
Issues here: https://github.com/adobe/react-spectrum/issues/850
and here: https://github.com/adobe/react-spectrum/issues/1155

**Reply by areknow** [reply]
Thank you @snowystinger 🙂
